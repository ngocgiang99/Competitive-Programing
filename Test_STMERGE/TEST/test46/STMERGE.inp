5
7 3
238346363 870674907 711148619 
242045861 619107789 102788746 
7732628 589234926 826690933 
45614010 149537549 603670746 
754485817 639092160 416192923 
112249823 227695388 852484914 
106101349 863857843 572931940 
8 10
666881609 881967767 26286059 564639355 621775319 257353337 270129053 311179091 916892882 549889295 
259805070 609527160 121001199 32323742 803873684 209240402 730394146 541396907 945867715 602850692 
72223906 627683067 207488112 985663732 882832098 549426068 601873457 666993350 266052505 586180794 
677855194 166831200 915571130 90403825 51776129 11942247 823173755 524702976 612992890 165874623 
529211729 874110955 776681050 381118862 126144497 161272056 704337143 48610105 735260240 816423594 
382661989 415572836 856247971 97907655 36823546 403376885 868426078 37088085 16644938 99747089 
325611832 367493784 595554346 428739356 735999268 671970581 589248432 453108924 18605145 834695825 
121165616 806541781 251169866 527735057 443866687 498043534 596748461 875956442 251113139 984363337 
7 4
145962144 972450990 940909277 428447388 
500159580 236192672 75360595 93855839 
876546171 501706564 922445200 236281599 
779121107 843879169 973314695 966253569 
864411511 349611642 715890031 345595178 
476273855 737144724 625515451 148595839 
349575997 881103990 771213089 179690566 
4 9
100761494 582075650 366506869 25772896 26987343 843808634 26712983 850810099 148754113 
878090930 607381446 197409633 936396449 350774767 208529108 919032846 668475876 440774326 
176551896 865004705 166993854 685147852 843718695 165987580 774447526 951177783 165249876 
490346311 521883471 473060154 745488694 224960997 954449071 893373532 810608610 482972539 
7 1
574898604 
73225968 
55724968 
561523790 
988897577 
363579751 
342115435 
