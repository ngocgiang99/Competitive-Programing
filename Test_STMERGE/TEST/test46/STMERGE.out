106101349
121165616
239817983
148754113
128950936
